js/Class/Basket.js

```diff
@@ -0,0 +1,152 @@
+      const wraper = document.createElement('div');
+      wraper.classList.add('idwraper')
+      wraper.setAttribute('idwraper', this.idWork);
+      doc.append(wraper);
+
+      const nameProduct = document.createElement('div');
+      nameProduct.innerHTML = `Наименование ${this.basketArr[this.idWork].product} цена ${this.basketArr[this.idWork].price} грн/шт<br>`
+      wraper.append(nameProduct);
+
+      const inputWraper = document.createElement('div');
+      inputWraper.classList.add('input-wraper');
+      wraper.append(inputWraper);
+
+      const inputMinus = document.createElement('input');
+      inputMinus.setAttribute('value', '-');
+      inputMinus.setAttribute('operation', 'minus');
+      inputMinus.setAttribute('type', 'button');
+      inputMinus.setAttribute('idinput', this.idWork);
+      inputMinus.classList.add('input-button');
+      inputWraper.append(inputMinus);
+      inputMinus.addEventListener('click', this.plusMinusGoods.bind(this))
+
+      const inputEntry = document.createElement('input');
+      inputEntry.setAttribute('idinput', this.idWork)
+      inputEntry.setAttribute('value', this.basketArr[this.idWork].amount)
+      inputEntry.setAttribute('type', 'number')
+      inputEntry.classList.add('input-count');
+      inputWraper.append(inputEntry);
+      inputEntry.addEventListener('input', this.changeAmountGoods.bind(this))
+
+
+      const inputPlus = document.createElement('input');
+      inputPlus.setAttribute('value', '+');
+      inputPlus.setAttribute('operation', 'plus');
+      inputPlus.setAttribute('type', 'button')
+      inputPlus.setAttribute('idinput', this.idWork)
+      inputPlus.classList.add('input-button');
+      inputWraper.append(inputPlus);
+      inputPlus.addEventListener('click', this.plusMinusGoods.bind(this))
+
+      const buttonDelete = document.createElement('div');
+      buttonDelete.innerHTML = `<button class="input-button" operation="del" ><span type="button" operation="del" idinput="${this.idWork}" 
+      class="material-symbols-outlined"> delete</span></button>`;
+      buttonDelete.setAttribute('operation', 'del');
+      buttonDelete.setAttribute('idinput', this.idWork)
+      inputWraper.append(buttonDelete);
+      buttonDelete.addEventListener('click', this.plusMinusGoods.bind(this))
+
+
+      const out = document.createElement('div');
+      inputWraper.append(out);
+      out.classList.add('out', this.idWork);
+      this.updateBasketAndCalculate();
+   }
+
+   updateBasketAndCalculate() {
+      let sumOrder = 0;
+      for (let key in this.basketArr) {
+         sumOrder += this.basketArr[key].price * this.basketArr[key].amount;
+      };
+      this.sumOrderTotal = sumOrder;
+
+      if (!this.deleteProductHTML) {
+         const doc = document.getElementsByClassName(this.idWork);
+         const arrElementInput = document.querySelectorAll('.input-count');
+         for (let key2 of arrElementInput) {
```

Comment:
never name variable with numbers, name should represents main idea of object /array's element